### PR TITLE
Fix tests after LVM API refactor

### PR DESCRIPTION
**Pull request by katlapinka**
Needs: 
https://github.com/Open-CAS/test-framework/pull/26
https://github.com/Open-CAS/test-framework/pull/25

### CODE REVIEW

File: test/functional/tests/volumes/common.py [hunk 1]
```diff
@@ -1,12 +1,19 @@
 #
 # Copyright(c) 2022 Intel Corporation
+# Copyright(c) 2024 Huawei Technologies Co., Ltd.
 # SPDX-License-Identifier: BSD-3-Clause
 #
+from api.cas.init_config import InitConfig, opencas_conf_path
```
Comment (Kamoppl) [inline test/functional/tests/volumes/common.py:6]: add empty line

File: test/functional/tests/volumes/test_many_lvms_on_many_cores.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
 #
 # Copyright(c) 2022 Intel Corporation
+# Copyright(c) 2024 Huawei Technologies Co., Ltd.
 # SPDX-License-Identifier: BSD-3-Clause
 #
```
Comment (Kamoppl) [inline test/functional/tests/volumes/test_many_lvms_on_many_cores.py:5]: add empty line

File: test/functional/tests/volumes/test_many_lvms_on_single_core.py [hunk 3]
```diff
@@ -1,19 +1,19 @@
 #
 # Copyright(c) 2022 Intel Corporation
+# Copyright(c) 2024 Huawei Technologies Co., Ltd.
 # SPDX-License-Identifier: BSD-3-Clause
 #
 import datetime
```
Comment (Kamoppl) [inline test/functional/tests/volumes/test_many_lvms_on_single_core.py:6]: as above

File: test/functional/tests/io_class/test_io_class_lvm_on_cas.py [hunk 4]
```diff
@@ -43,16 +45,14 @@ def test_io_class_lvm_on_cas():
         cache = casadm.start_cache(cache_dev.partitions[0], CacheMode.WB, force=True)
         core = cache.add_core(core_dev.partitions[0])
 
-    with TestRun.step("Create LVM on CAS device."):
-        lvm_filters = ["a/.*/", "r|/dev/sd*|", "r|/dev/hd*|", "r|/dev/xvd*|", "r/disk/", "r/block/",
-                       "r|/dev/nvme*|"]
+    with TestRun.step("Add CAS device type to the LVM config file."):
```
[the remark this answers is not in the document]
Comment (katlapinka) [reply test/functional/tests/io_class/test_io_class_lvm_on_cas.py:48]: as we talked - this step is not needed because test doesn't reboot the platform

**Review by Deixx — APPROVED** [review]

**Review by Kamoppl — APPROVED** [review]

**Pull request merged**